build.sbt

```diff
@@ -39,7 +39,7 @@ lazy val commonSettings = Seq(
   bintrayOrganization := Some("buildo"),
   organization := "io.buildo",
   licenses += ("MIT", url("https://github.com/buildo/wiro/blob/master/LICENSE")),
-  scalaVersion := "2.11.8",
+  scalaVersion := "2.12.4",
```

Comment:
2.12.5 ;)

Reply:
Hm... it looks like it breaks things up. Not sure why.
Merging this for the moment :P